### PR TITLE
Multigpu val

### PR DESCRIPTION
**Pull request by chuckcho**
properly average val loss

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -200,8 +199,8 @@ def get_true_counts(tower_logits_labels):
             tf.Summary.Value(tag="val_acc", simple_value=acc)
           ])
           summary_writer.add_summary(acc_summ, it)
-          # add val loss to summary
-          val_loss = np.mean(val_loss)
+          # a dd val loss to summary
+          val_loss = val_loss / int(len(val_pipeline.videos) / NUM_GPUS / BATCH_SIZE)
```
Comment (Peilin-D) [inline train.py]: Thanks! This is correct. But then the previously calculated loss_val would be much larger. I wonder why it still looked fine on tensorboard? Did you correct this when running the experiments?

Comment (chuckcho) [reply train.py]: yes. i noticed at the log, val_loss was a lot larger than train_loss.
